packages/contractkit/package.json

```diff
@@ -40,6 +40,7 @@
     "web3-utils": "1.0.0-beta.37"
   },
   "devDependencies": {
+    "@celo/dev-utils": "0.0.1",
```

Comment:
Can you make this version `-dev` to keep with the new scheme @mcortesi just introduced?

Reply:
Will do, though I imagine this is not a package we ever have to publish to npm i think?